src/components/accessible-svg/AccessibleSVG.js.flow

```diff
@@ -0,0 +1,41 @@
+// @flow
```

Comment:
I don't think these .flow files were checked in before

Reply:
These flow files are to keep VSCode happy with flow+ts together while you are working within BUIE. Its similar to the ones we generated at publish time because at that time it was only for consuming projects. These ones are for BUIE now.